src/System.IO.Pipelines/src/System.IO.Pipelines.csproj

```diff
@@ -38,11 +36,11 @@
   <ItemGroup Condition="'$(TargetGroup)'=='netcoreapp'">
     <Compile Include="System\IO\Pipelines\ThreadPoolScheduler.NetCoreApp21.cs" />
```

Comment:
And this one should be `.netcoreapp.cs`

Reply:
No version?

Reply:
Right. netcoreapp in corefx means current netcoreapp version.